### ISSUE TITLE
Prisma plugin & Cloudflare workers

### ISSUE THREAD

**Issue by kfields** [hayes/pothos#623]
Hi!  I've been experimenting with Pothos and Prisma on Node and loving it.  Cloudflare workers however is a bit of a challenge due to the lack of globals.  This works and I suppose it's not *too* bad:
```typescript
import { createYoga } from 'graphql-yoga'
import { Settings, configure } from "@gspacex/lib-settings";

export default {
  async fetch(request: Request, env: Settings, ctx: ExecutionContext) {
    configure(env);
    const { schema } = await import("@gspacex/lib-graphql");
    const yoga = createYoga<{ env: Env; ctx: ExecutionContext }>({
      schema,
    });
    return yoga.handleRequest(request, { env, ctx });
  },
};
```
Next I tried to lazily create the prisma client in the builder and in the resolvers so I could do this instead:
```typescript
import { createYoga } from "graphql-yoga";
import { Settings, configure } from "@gspacex/lib-settings";
import { schema } from "@gspacex/lib-graphql";

const yoga = createYoga<{ env: Env; ctx: ExecutionContext }>({
  schema,
});

export default {
  async fetch(request: Request, env: Settings, ctx: ExecutionContext) {
    configure(env);
    return yoga.handleRequest(request, { env, ctx });
  },
};
```
However it results in this error:
```bash
/home/kurt/Dev/gspacex/node_modules/@pothos/core/src/utils/context-cache.ts:20
    const cacheKey = (context as { [contextCacheSymbol]: object })[contextCacheSymbol] || context;
                      ^
TypeError: Cannot read properties of undefined (reading 'Symbol(Pothos.contextCache)')
    at getRelationMap (/home/kurt/Dev/gspacex/node_modules/@pothos/core/src/utils/context-cache.ts:20:23)
    at SchemaBuilder.prismaObject (/home/kurt/Dev/gspacex/node_modules/@pothos/plugin-prisma/src/schema-builder.ts:29:20)
    at /home/kurt/Dev/gspacex/libs/graphql/src/schema/core.ts:5:9
    at SourceTextModule.evaluate (node:internal/vm/module:224:23)
    at VMScriptRunner.runAsModule (/home/kurt/Dev/gspacex/node_modules/@miniflare/runner-vm/src/index.ts:38:18)
    at VMScriptRunner.run (/home/kurt/Dev/gspacex/node_modules/@miniflare/runner-vm/src/index.ts:82:17)
    at EventTarget.#reload (/home/kurt/Dev/gspacex/node_modules/@miniflare/core/src/index.ts:775:13)
```
What's odd is that it does work under Node using the same libraries:
```typescript
import * as dotenv from 'dotenv'
dotenv.config()

import { createServer } from 'http'
import { createYoga } from 'graphql-yoga'

import { configure } from "@gspacex/lib-settings";
configure(process.env)

import { schema } from '@gspacex/lib-graphql'

const yoga = createYoga({
  schema,
});
const server = createServer(yoga)

server.listen(4000, '127.0.0.1')
```
Any help or a workaround is appreciated.  Thanks!

**Comment by kfields** [comment]
Never mind, it turned out Prisma.dmmf  that I was passing to the SchemaBuilder constructor was undefined because I was pulling it from the wrong library.  This fixed it:
```typescript
//import { Prisma } from '@prisma/client';
import { Prisma } from '@prisma/client/edge';
``` 
Keep up the good work!!!

**Comment by arimgibson** [comment]
@kfields were you able to get Pothos, Prisma, and CF Workers to play nicely together? Can't seem to get past errors saying that I've exceeded the script startup time. I've tried importing the schema inside the handler but it comes back without any mutations. Curious what your code looks like if it all worked!

**Comment by kfields** [comment]
@arimgibson , yeah it works but the code might be hard to follow since I made it where it can run under either Node or Workers.
Here's the repo: https://github.com/gspacex/gspacex
Email me if you have any questions

**Comment by arimgibson** [comment]
You're the best @kfields; really appreciate it! I had tried to google the package you were importing from the find the code on my own but didn't have any luck. Reading over now; you might've saved me a lot more headache haha
